librz/analysis/esil_dfg.c

```diff
@@ -179,24 +179,28 @@ static int _rv_ins_cmp(void *incoming, void *in, void *user) {
 	return rv_incoming->from - rv_in->from;
 }
 
+static EsilDFGRegVar *newEsilDFGRegVar(const RzRegItem *ri, RzGraphNode *node) {
+	EsilDFGRegVar *rv = RZ_NEW0(EsilDFGRegVar);
+	if (!rv) {
+		return NULL;
+	}
+	rv->from = ri->offset;
```

Comment:
Kudos to you for understanding how those shift/or/xor mapped to this :D Did you double check they are really the same? Honestly, I couldn't figure that out :/